python-package/lightgbm/dask.py

```diff
@@ -626,7 +660,14 @@ def predict_proba(self, X: _DaskMatrixLike, **kwargs: Any) -> dask_Array:
             **kwargs
         )
 
-    predict_proba.__doc__ = LGBMClassifier.predict_proba.__doc__
+    predict_proba.__doc__ = _lgbmmodel_doc_predict.format(
+        description="Return the predicted probability for each class for each sample.",
+        X_shape="dask Array or dask DataFrame of shape = [n_samples, n_features]",
+        output_name="predicted_probability",
+        predicted_result_shape="dask Array of shape = [n_samples] or shape = [n_samples, n_classes]",
```

Comment:
Not sure, but guess it is the same as for sklearn.
```suggestion
        predicted_result_shape="dask Array of shape = [n_samples, n_classes]",
```

Reply:
Oh thank you, my mistake. I thought I remembered that there was an inconsistency in the case of binary classification, where `predict_proba()` only returned an array of shape `[n_samples]`.

I was thinking about https://github.com/microsoft/LightGBM/issues/3636, but I guess I remembered it incorrectly.